Cargo.toml

```diff
@@ -577,6 +577,768 @@ serde_yaml = { version = "0.9" }
 syn = { version = "2.0.53" }
 thiserror = { version = "1.0.48" }
 tracing-subscriber = { version = "0.3.18" }
+Inflector = { version = "0.11.4" }
```

Comment:
how about grouping them by internal and external (path and version)?

Reply:
Yea we could think about groupings, but personally i just put it in here and use the taplo to format them.  
If taplo does not enforce the order, then someone will mess it up inevitably.